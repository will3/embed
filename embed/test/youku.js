const test = require('./utils/test');

test('youku', 
	'http://v.youku.com/v_show/id_XMjcyMTA3MjM5Ng',
	'http://player.youku.com/embed/XMjcyMTA3MjM5Ng==');

// .todo
// vku.youku.com/live/play?id=12137