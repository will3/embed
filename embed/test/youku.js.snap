// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` youku 1`] = `
Object {
  "originalUrl": "http://v.youku.com/v_show/id_XMjcyMTA3MjM5Ng",
  "site_name": undefined,
  "title": "御姐归来 01—在线播放—《御姐归来》—电视剧—优酷网，视频高清在线观看",
  "video": Object {
    "description": undefined,
    "height": "498",
    "title": undefined,
    "url": "http://player.youku.com/embed/XMjcyMTA3MjM5Ng==",
    "width": "510",
  },
}
`;
